monkey/common/network/network_range.py

```diff
@@ -81,7 +81,7 @@ def check_if_range(address_str):
         if -1 != address_str.find("-"):
             try:
                 NetworkRange._range_to_ips(address_str)
-            except ValueError:
+            except (ValueError, OSError):
```

Comment:
This isn't a good fix. An `OSError` could be raised for any number of reasons.

Reply:
The solution to this should be to validate that a particular input is a hostname and then proceed.